akka-remote/src/main/scala/akka/remote/artery/ObjectPool.scala

```diff
@@ -0,0 +1,24 @@
+/**
+ * Copyright (C) 2016 Lightbend Inc. <http://www.lightbend.com>
+ */
+package akka.remote.artery
+
+import org.agrona.concurrent.ManyToManyConcurrentArrayQueue
```

Comment:
but then we need to remember to update that when we update Aeron, what would the benefit be of an explicit dependency? Perhaps when we release something final, but right now it will only be a risk of mismatch.